### PR TITLE
IGNITE-24497 Rename docker image in docs

### PR DESCRIPTION
**Pull request by IgGusev**
Thank you for submitting the pull request.

To streamline the review process of the patch and ensure better code quality
we ask both an author and a reviewer to verify the following:

### The Review Checklist
- [ ] **Formal criteria:** TC status, codestyle, mandatory documentation. Also make sure to complete the following:  
\- There is a single JIRA ticket related to the pull request.  
\- The web-link to the pull request is attached to the JIRA ticket.  
\- The JIRA ticket has the Patch Available state.  
\- The description of the JIRA ticket explains WHAT was made, WHY and HOW.  
\- The pull request title is treated as the final commit message. The following pattern must be used: IGNITE-XXXX Change summary where XXXX - number of JIRA issue.
- [ ] **Design:** new code conforms with the design principles of the components it is added to.
- [ ] **Patch quality:** patch cannot be split into smaller pieces, its size must be reasonable.
- [ ] **Code quality:** code is clean and readable, necessary developer documentation is added if needed.
- [ ] **Tests code quality:** test set covers positive/negative scenarios, happy/edge cases. Tests are effective in terms of execution time and resources.

### Notes
- [Apache Ignite Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Java+Code+Style+Guide)

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -25,15 +25,15 @@ The following examples are included:
 IGNITE_SOURCES=/path/to/ignite3-sources-dir
 ```
 
-3. Build the Ignite Docker image. As a result the `apacheignite/ignite3` image will be built and loaded into the Docker Engine:
+3. Build the Ignite Docker image. As a result the `apacheignite/ignite` image will be built and loaded into the Docker Engine:
```
Comment (ptupitsyn) [inline examples/README.md:28]: You don't have to build it yourself, let's remove this step - we publish the image as part of the release.

File: packaging/build.gradle [hunk 2]
```diff
@@ -82,7 +82,7 @@ def replaceVars = tasks.register('replaceVars', Copy) {
 }
 
 docker {
-    name 'apacheignite/ignite3'
+    name 'apacheignite/ignite:3.0.0'
```
[the remark this answers is not in the document]
Comment (IgGusev) [reply packaging/build.gradle:85]: This would pull latest, which is currently set to 2.17. As we are sharing docker repos, we cannot not hard code this is we want Ignite 3.

File: packaging/docker/docker-compose.yml [hunk 3]
```diff
@@ -19,7 +19,7 @@ name: ignite3
 
 x-ignite-def:
   &ignite-def
-  image: apacheignite/ignite3:${IGNITE3_VERSION:-latest}
+  image: apacheignite/ignite:3.0.0
```
[the remark this answers is not in the document]
Comment (IgGusev) [reply packaging/docker/docker-compose.yml:22]: There is no apacheignite/ignite3 unless i am missing something.

[the remark this answers is not in the document]
Comment (ptupitsyn) [reply packaging/docker/docker-compose.yml:22]: Fixed the suggestion.

File: packaging/build.gradle (hunk 2 above)
[the remark this answers is not in the document]
Comment (ptupitsyn) [reply packaging/build.gradle:85]: This is script that builds a new image. It does not pull existing images. It will be fixed in https://issues.apache.org/jira/browse/IGNITE-24408

File: examples/README.md [hunk 4]
```diff
@@ -18,38 +18,31 @@ The following examples are included:
...
-4. Start an Ignite node:
+2. Start an Ignite node:
 ```shell
 docker run --name ignite3-node -d --rm -p 10300:10300 -p 10800:10800 \
-  -v $IGNITE_SOURCES/examples/config/ignite-config.conf:/opt/ignite/etc/ignite-config.conf apacheignite/ignite3
+  -v $IGNITE_SOURCES/examples/config/ignite-config.conf:/opt/ignite/etc/ignite-config.conf apacheignite/ignite:3.0.0
 ```
 
-5. Get the IP address of the node:
+3Get the IP address of the node:
```
Comment (ptupitsyn) [inline examples/README.md:32]:
```suggestion
3. Get the IP address of the node:
```

**Review by ptupitsyn — APPROVED** [review]
